lib/blumquist.rb

```diff
@@ -51,6 +55,13 @@ def resolve_json_pointers
     end
   end
 
+  def resolve_json_pointer(type_def)
```

Comment:
I looked again into this: When you set the type for the `oneOf`([here](https://github.com/moviepilot/blumquist/pull/17/files#diff-cb52284b877d325638c7b0cefe622506R197)), "one" would be already resolved, so instead of having what you need `{"$ref" => "#/definitions/foo" }` you have the type referenced.

Alternatives are, make a copy of `one` before resolving it, or naming `resolve_json_pointer` differently, as...it's not resolving them :).